packages/flutter_test/lib/src/binding.dart

```diff
@@ -700,13 +751,43 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
     // The LiveTestWidgetsFlutterBinding overrides this to report the exception to the console.
   }
 
+  Future<dynamic> _handleMessage(dynamic mockMessage) async {
```

Comment:
`Future<void>` as return type and `Object? mockMessage` as argument?

Reply:
Good point! Done!